site-src/geps/gep-713.md

```diff
@@ -0,0 +1,496 @@
+# GEP-713: Policy Attachment
+
+* Issue URL: https://github.com/kubernetes-sigs/gateway-api/issues/713
+* Status: Implementable
+
+## TLDR
+
+This GEP aims to standardize policy attachment to resources associated with
+Gateway API. This will be important for providing a consistent experience across
+implementations of the API, even for configuration details that may not be fully
+portable.
+
+## Goals
+
+* Establish a pattern for policy attachment which will be used for any policies
+  included in the Gateway API spec
+* Establish a pattern for policy attachment which should be used for any
+  implementation specific policies used with Gateway API resources
+* Provide a way to distinguish between required and default values
+* Enable policy attachment at all relevant scopes, including Gateways, Routes,
+  Backends, and maybe Namespaces
+* Provide a means of attachment that works for both ingress and mesh uses of
+  this API
+* Provide a consistent way that both included and implementation-specific
+  policies can be interpreted
+
+## Out of scope
+
+* Define all potential policies that may be attached to resources
+* Design the full structure and configuration of policies
+
+## API
```

Comment:
I think this document needs a section here that describes what a Policy object is. It seems like they have the following properties:
- Can be an object of any group or kind
- Must have a `targetRef` field, as described below.

I actually think it's almost worth formalizing that a Policy object is an object that will decode into a (provided) skeleton struct that only has the `targetRef` field (along with the usual object fields).

Reply:
This skeleton should also include the presence of `default` and `override` fields, with an explanation of what they do. I think that providing an expected skeleton will make building new, compliant Policy objects much clearer.

Reply:
+1 to this one.

Reply:
Great idea, added a sample policy resource to the GEP.